packages/nimble-components/src/label-provider/rich-text-editor/label-token-defaults.ts

```diff
@@ -0,0 +1,12 @@
+import type * as TokensNamespace from './label-tokens';
+
+type TokenName = keyof typeof TokensNamespace;
+
+export const richTextEditorLabelDefaults: {
+    readonly [key in TokenName]: string;
+} = {
+    richTextEditorToggleBoldLabel: 'Bold',
+    richTextEditorToggleItalicsLabel: 'Italics',
+    richTextEditorToggleBulletListLabel: 'Bullet List',
```

Comment:
I checked a couple other products like GitHub and AzDO and they seem to prefer this language, which aligns better with "Numbered List".

Ideally we'd also update the names of the properties to match.

```suggestion
    richTextEditorToggleBulletListLabel: 'Bulleted List',
```

Reply:
Please do it uniformly for all these properties, token names, element attributes, etc, not just for the string as shown here.

```suggestion
    richTextEditorToggleBulletedListLabel: 'Bulleted List',
```

Reply:
Updated